### ISSUE TITLE
Where are my new items?

### ISSUE THREAD

**Issue by nfbauti** [joomla/joomla-cms#36616]
### Steps to reproduce the issue

Where are my new items?

Cordial greeting.

On January 7 update joomla 3.10.4 to 4.0.5 the process was carried out normally and at the moment it works well and that same day (January 7) I did not make any article.

However, the new articles do not appear listed in the backend but in the frontend they do appear, that is, I can create and save them but they do not appear listed with the other articles, only those created until January 6 appear (screenshot attached).

Did anyone have that problem? Is there a solution?

However, if I enter the database and check the content table if the articles created on January 8 appear (attached screenshot)

https://portalnews.co

Thank you all very much for your help.
![listado backend](https://user-images.githubusercontent.com/33980599/148680894-4365e1b5-4363-4b39-b27c-bfcb5ac2c78d.jpg)
![tabla content](https://user-images.githubusercontent.com/33980599/148680895-b631c95d-2944-4cd8-b62f-0f6b5e98895e.jpg)

**Comment by brianteeman** [comment]
How are you creating the articles

**Comment by nfbauti** [comment]
They are created with the article manager and the JCE plugin, the system saves them but they are not listed in the article manager, the problem started after migrating from joomla 3.10.04 to 4.0.5, otherwise Joomla works correctly.

I use php 7.4 and Helix Ultimate template<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36616">issues.joomla.org/tracker/joomla-cms/36616</a>.</sub>

**Comment by brianteeman** [comment]
In the database please check and see if there is an entry in the workflows_associations table for each of the _missing_ articles

**Comment by nfbauti** [comment]
I have verified the table workflows_associations and it does not contain any records, it is blank
![w47fa_associations](https://user-images.githubusercontent.com/33980599/148682806-d2a48a36-7c7b-4c3c-b51f-9541de32f422.jpg)

**Comment by brianteeman** [comment]
ok so now we know that is the problem.

the question is how that has happened if you are creating the articles in the admin using com_content.

we've seen it happen with some import extensions that are not j4 compatible but I've never seen it when using com_content.

As a test could you go to global configuration and change the default editor to none and then create a new article and see if it is visible

**Comment by nfbauti** [comment]
When doing what is requested and saving the article, both published and unpublished, the following error appears:

0 Class name must be a valid object or a string

the same thing happens with the TinyMCE editor
![sin editor](https://user-images.githubusercontent.com/33980599/148683053-f6037932-9e88-4292-92b8-6872be09938b.jpg)

**Comment by brianteeman** [comment]
And that doesn't happen when you do exactly the same with JCE?

Now enable debug in global configuration and repeat the tests. This time you should get a stack trace of the error

**Comment by nfbauti** [comment]
Before changing the editor in global settings, JCE created and saved new articles but they were not visible in the article manager.

When changing it to TinyMCE I generated the error "0 Class name must be a valid object or a string" and when I changed it back to JCE I also generated the same error "0 Class name must be a valid object or a string"

When I publish a new article and enter it with the debugger enabled, this comes out:

Now I can't find how to delete those test articles created

https://portalnews.co/cundinamarca/test-2

![debugpage](https://user-images.githubusercontent.com/33980599/148683634-63012827-6b7a-48fe-9796-d4ef64666f0c.jpg)

**Comment by brianteeman** [comment]
That's an error message in the front end - I need to see the backend error message as the stack trace will be different

Please follow these exact steps and report any errors at each stage

1. TinyMCE as the editor and Debug on in the global configuration.
2. Create a new article in the backend and save
3. Display the article in the frontend.

Also please go to the extension manager and set the filters as shown below and report if there are any content and system plugins.

![image](https://user-images.githubusercontent.com/1296369/148684169-a6342e96-4d53-4717-bc20-3f2ba6b3c3b2.png)

![image](https://user-images.githubusercontent.com/1296369/148684191-3fa473fc-c16f-4448-86ac-70025ebfacdf.png)

**Comment by nfbauti** [comment]
I did the test as requested and this came out
![x](https://user-images.githubusercontent.com/33980599/148684523-064035f5-024b-4352-9e8d-a0553ff44936.jpg)
![y](https://user-images.githubusercontent.com/33980599/148684524-bc2b54a0-3def-4817-be7a-c14d6e176afe.jpg)

**Comment by brianteeman** [comment]
So that confirms that the problem is coming from the plugin called pingomatic. Disable that plugin and it should be fine

**Comment by nfbauti** [comment]
Pingomatic is part of the sitemap plugin called JSitemap

I have disabled the pingomatic and all the JSitemap extension

Clear Joomla and Chrome cache

Check the list of articles in the backend and it remains the same, the new articles created do not appear but in the database table content if you register them, the problem continues.

**Comment by brianteeman** [comment]
and for articles you create now  that pingomatic is disabled?

**Comment by nfbauti** [comment]
Same, pingomatic enabled or disabled generates the same.

New items are not listed.

If you want and in an incredible act of trust only with you, I could create a clone of my website and give it credentials to do all the necessary tests without limits and once the error is detected, tell me what it is and then I will proceed to eliminate the testing website.

I would need a confidential email from you to not reveal the data in this post, keep in mind that this is not an easy decision on my part, but I understand that it is the best way for you to work more easily.

I remain pending your decision.

**Comment by brianteeman** [comment]
I will decline that offer.

**Comment by nfbauti** [comment]
Okay, so let's keep trying to find the error hereby exchanging messages

**Comment by brianteeman** [comment]
its 100% an extension causing this and not core.

**Comment by nfbauti** [comment]
Uninstall the extensions and plugins that are not part of joomla and I will test.

**Comment by nfbauti** [comment]
I am totally perplexed, in the test site I uninstalled and I was installing add-ons and plugins and in the end they all worked correctly and when creating new articles they are now visible in the general list of articles and I can edit them and save their changes with JCE

What I did then in the real site was to reinstall the following components and plugins and as far as I can see, the list of articles already works, the new ones are seen and I can edit them.

List of components and plugins in this test:

Akeeba Backup
EngageBox
Fast Social Share
Inner Content Pro
JCE
JSitemap PRO
Phoca Open Graph Plugin
SP Page Builder
Sp Polls (installed and disabled)
SP Cookie Consent (installed and disabled)

In any case, I greatly appreciate your collaboration and I hope you have not taken the wrong person who wanted to provide you with trial access to my website, I did not do it with bad intentions, I just thought it would be easier if you entered directly and looked at what you as a programmer it would be more useful to solve the problem.

Thank you very much for everything and we will see you another time.
